COMETwebapp/COMETwebapp/Components/OpenModel.razor

```diff
@@ -0,0 +1,194 @@
+﻿<!------------------------------------------------------------------------------
+Copyright (c) 2022 RHEA System S.A.
+
+    Authors: Justine Veirier d'aiguebonne, Sam Gerené, Alex Vorobiev, Alexander van Delft
+
+    This file is part of COMET WEB Community Edition
+     The COMET WEB Community Edition is the RHEA Web Application implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+
+    The COMET WEB Community Edition is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Affero General Public
+    License as published by the Free Software Foundation; either
+    version 3 of the License, or (at your option) any later version.
+
+    The COMET WEB Community Edition is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see http://www.gnu.org/licenses/.
+------------------------------------------------------------------------------->
+
+@inject ISessionAnchor ISessionAnchor 
+@inject NavigationManager NavigationManager
+
+@if (ModelSetupList.Count == 0)
+{
+    InitializeModelList();
+}
+
+<div class="form-group row">
+    <label>Model:</label>
+    <div class="col-md-15 m-top-3px">
+        <select @onchange="SetNameModel">
+            <option value="">--Select--</option>
+            @foreach (var model in ModelSetupList)
+            {
+                <option value="@model.Name">@model.Name</option>
+            }
+        </select>
+    </div>
+</div>
+
+@if (IterationSetupList.Count != 0)
+{
+   <div class="form-group row">
+    <label>Iteration:</label>
+    <div class="col-md-15 m-top-3px">
+        <select @onchange="SetIteration">
+            <option value="">--Select--</option>
+            @foreach(var iteration in IterationSetupList)
+            {
+                <option value="@iteration.Description">@iteration.Description</option>
```

Comment:
perhaps here we can use `iteration number - date`?  please have a look at the IME?

Reply:
indeed